source/proof_resolution_mock/lib/state_id_mock_client.rb

```diff
@@ -19,6 +19,8 @@ class StateIdMockClient < Proofer::Base
       drivers_license drivers_permit state_id_card
     ].to_set.freeze
 
+    INVALID_STATE_ID_NUMBER = '00000000'.freeze
```

Comment:
just as a note, we check for any "all zeroes" state ID to fail, instead of checking this exact constant, which I think is OK